### PR TITLE
Wrapping terrain info

### PR DESCRIPTION
**Pull request by Night-Pryanik**
Before: first string is too long, so it tries to go the second string. Movement cost isn't shown.
Third string is too long, so `Mountable.` text isn't shown either.
default

After: long strings are wrapped gracefully.
default

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -7744,28 +7744,31 @@ void game::print_visibility_info( const catacurses::window &w_look, int column,
...
-        mvwprintw(w_look, line, column, _("%s; Impassable"), tile.c_str());
+    if( m.impassable( lp ) ) {
+        lines = fold_and_print( w_look, line, column, max_width, c_dark_gray, _( "%s; Impassable" ),
+                                tile.c_str() );
     } else {
-        mvwprintw(w_look, line, column, _("%s; Movement cost %d"), tile.c_str(),
-                  m.move_cost(lp) * 50);
+        lines = fold_and_print( w_look, line, column, max_width, c_dark_gray, _( "%s; Movement cost %d" ),
+                                tile.c_str(),
+                                m.move_cost( lp ) * 50 );
```
Comment (illi-kun) [inline src/game.cpp]: weird formatting here

Comment (Night-Pryanik) [reply src/game.cpp]: That's astyle's formatting.

Comment (illi-kun) [reply src/game.cpp]: There are more than one way to pass astyle check; this formatting also passed the check and looks simpler:
```
        lines = fold_and_print( w_look, line, column, max_width, c_dark_gray, _( "%s; Movement cost %d" ),
                                tile.c_str(), m.move_cost( lp ) * 50 );
```

**Comment by kevingranade** [comment]
Spawn mjolnir, walk outside the shelter, smash a window, examine the tile.

Drpped items overlap some of the terrain text.

**Pull request merged**